terraform/backend_uk.tf

```
@@ -0,0 +1,55 @@
+module "backend_uk" {
+  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/ecs-service?ref=aws/ecs-service-v1.5.0"
+
+  service_name  = var.service_name
+  service_count = var.service_count
+  environment   = var.environment
+  region        = var.region
+
+  cluster_name     = "trade-tariff-cluster-${var.environment}"
+  subnet_ids       = data.aws_subnets.private.ids
+  security_groups  = [data.aws_security_group.this.arn]
+  target_group_arn = data.aws_lb_target_group.this.arn
+
+  min_capacity = var.min_capacity
+  max_capacity = var.max_capacity
+  docker_image = data.aws_ssm_parameter.ecr_url.value
+  docker_tag   = var.docker_tag
+  skip_destroy = true
+
+  cloudwatch_log_group_name = data.aws_cloudwatch_log_groups.log_group.log_group_names
+
+  service_environment_config = [
```

```suggestion
  service_environment_config = flatten(
    local.backend_common_vars,
    [
```

We'll want to add the common list of vars and flatten the array down.